catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -132,11 +132,16 @@ public CreateResponse create(
     CreateResponse createResponse = null;
     CreateStorageRequest createStorageRequest = null;
     CreateStorageResponse createStorageResponse;
+    CreateRequest createRequest = null;
+    Exception ingestError = null;
 
     streamCreateRequest =
         opsStorageSupport.prepareStorageRequest(
             streamCreateRequest, streamCreateRequest::getContentItems);
 
+    INGEST_LOGGER.info(
+        "Resource ingest has started for resource id: {}.", streamCreateRequest.getId());
```

Comment:
✏️ the phrase "Resource ingest" reads a little clunky from a user-perspective. Something like "Started ingesting resource with title {}". (This applies to all the instances of "Resource ingest" and "metacard ingest" in here)

Reply:
fixed